tests/integration-tests/tests/test_fsx_lustre.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+import re
+
+import boto3
+import pytest
+from retrying import retry
+
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutor
+from time_utils import minutes, seconds
+
+
+@pytest.mark.regions(["us-east-1", "eu-west-1"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.oss(["centos7"])
+@pytest.mark.schedulers(["sge"])
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+def test_lustre(region, pcluster_config_reader, clusters_factory, s3_bucket_factory, test_datadir):
```

Comment:
please rename to `test_fsx_lustre` or add an `fsx` marker

Reply:
Renamed!